dbms/src/Storages/DeltaMerge/Index/VectorIndexHNSW/Index.cpp

```diff
@@ -142,6 +142,18 @@ void VectorIndexHNSWBuilder::save(std::string_view path) const
     RUNTIME_CHECK_MSG(result, "Failed to save vector index: {} path={}", result.error.what(), path);
 }
 
+void VectorIndexHNSWBuilder::save(WriteBuffer & write_buf) const
```

Comment:
`saveToBuffer` may be more clear. I am not sure whether the compiler will implicitly cast `string_view` as `WriteBuffer` cause by the future code changes.

Reply:
Addressed.